src/CommandingOfficers/Commander.java

```diff
@@ -50,8 +50,9 @@ public class Commander implements GameEventListener, Serializable, UnitModifierW
   public static final int CHARGERATIO_HP = 100; // Funds value of 10 HP damage dealt, for the purpose of power charge
   public int incomeAdjustment = 0; // Commander subclasses can increase/decrease income if needed.
   private int myAbilityPower = 0;
+  public boolean roundUpRepairs = true; // I blame AWBW
```

Comment:
Does AWBW not? That sounds like a bug on them.

Reply:
It's been known for years. They ain't fixing it. Same deal with the way unloading works (which I may or may not try to accommodate)

Reply:
Is their unloading broken too?

Reply:
Yes. Unloading can be done by waited units, and doesn't take an action.
An APC can load, unload, move, resupply, load, and unload again all in one turn.
A Cruiser can kill a copter and then unload a copter into its space.

Reply:
OP